### ISSUE TITLE
Doesn't work when the schema is too large

### ISSUE THREAD

**Issue by wingzx3** [microsoft/copilot-pg#22]
When the database schema is too large it copilot errors complaining about the length of the message.

**Comment by robconery** [comment]
Yeah that's an issue with the API. Hopefully going to have a tweak to the API in the coming months. Leaving this open for now as there should be a way to do a workaround.

**Comment by hmorneau** [comment]
@wingzx3 Is it still an issue, it seems like Copilot chat accept much longer input now.
